static/app/views/alerts/list/rules/alertRulesList.spec.tsx

```diff
@@ -446,4 +448,41 @@ describe('AlertRulesList', () => {
       })
     );
   });
+
+  it('does not display ACTIVATED Metric Alerts', async () => {
+    rulesMock = MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/combined-rules/',
+      headers: {Link: pageLinks},
+      body: [
+        ProjectAlertRuleFixture({
```

Comment:
why make an issue alert rule that isn't checked for in this test?

Reply:
I copy pasta'd - 😿